src/Paysera/WalletApi/Entity/Statement.php

```diff
@@ -12,6 +12,9 @@ class Paysera_WalletApi_Entity_Statement
     const TYPE_TAX = 'tax';
     const TYPE_RETURN = 'return';
 
+    const DIRECTION_CREDIT = 'in';
```

Comment:
I think it would be more clearer and more consistent with API specs if we would use corresponding constant names (DIRECTION_IN and DIRECTION_OUT)